src/calculateRentalCost.js

```diff
@@ -1,10 +1,20 @@
+'use strict';
+
 /**
  * @param {number} days
- *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50; 
+  } else if (days >= 3) {
+    totalCost -= 20; 
```

Comment:
Similarly, use a constant for the short term rental threshold and discount. For instance, `const SHORT_TERM_RENTAL_THRESHOLD = 3;` and `const SHORT_TERM_RENTAL_DISCOUNT = 20;`.